includes/CAPx/Drupal/Mapper/EntityMapper.php

```diff
@@ -263,6 +296,56 @@ public function mapFieldCollections($data) {
   }
 
   /**
+   * Process entity reference fields uniquely.
+   *
+   * Reference fields are a special field and need to be handled differently.
+   * Allow for the ability to look for an item to attach to from the API
+   * that has already been imported.
+   *
+   */
+  public function mapReferences() {
```

Comment:
References act just like field collections in this case. CAPx does not require field collections either. This method is for handling the fields when they are available. When they are not available the references config setting will be an empty array.